packages/quantic/force-app/main/default/lwc/quanticCitation/quanticCitation.js

```diff
@@ -42,6 +49,14 @@ export default class QuanticCitation extends NavigationMixin(LightningElement) {
   isInitialRender = true;
   /** @type {string} */
   salesforceRecordUrl;
+  /** @type {boolean} */
+  isWithTextFragment = false;
+
+  connectedCallback() {
+    const fileType = this.citation?.fields?.filetype;
+    this.isWithTextFragment =
```

Comment:
This I think should be renamed because when I read `isWithTextFragment` I don't  easily understand what's going on,  what do you think of `this.isTextFragmentEnabled`

Reply:
idk about enabled since its not really something to be enabled/disabled. We could however improve it with something like `isHrefWithTextFragment` or something like that